css/css-fonts/parsing/font-family-valid.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Fonts Level 3: parsing font-family with valid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://drafts.csswg.org/css-fonts/#font-family-prop">
+<meta name="assert" content="font-family supports the full grammar '[ <family-name> | <generic-family> ] #'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_valid_value("font-family", "Helvetica");
+test_valid_value("font-family", "Helvetica, Verdana");
+
+test_valid_value("font-family", "serif");
+test_valid_value("font-family", "serif, sans-serif, cursive, fantasy, monospace");
```

Comment:
Not a blocker, but would be great to see a related test for Fonts 4 with the larger list of generic families

Reply:
People familiar with CSS Fonts 4 can add test cases as a later PR.

Other comments have been addressed.